### PR TITLE
veronoi function refactored

### PR DESCRIPTION
**Pull request by maxm17**
veronoi function refactored. Naming of inner functions need to be checked

### CODE REVIEW

File: resqpy/olio/triangulation.py [hunk 1]
```diff
@@ -658,7 +676,7 @@ def triangulated_polygons(p, v, centres = None):
     t_index = 0
 
     for cell, poly_vertices in enumerate(v):
-        # add polygon centre points to points array, if not already provided
+        # add polygon centre points to point array, if not already provided
```
Comment (cflynn3) [inline resqpy/olio/triangulation.py:679]: I think 'points' was correct, as in an array of points?

**Review by emmanesbit — APPROVED** [review]
Looks good to me

File: resqpy/olio/triangulation.py (hunk 1 above)
Comment (andy-beer) [reply resqpy/olio/triangulation.py]: yes, 'points' is better

**Comment by andy-beer** [comment]
I see that the branch has been merged into master.  Did we check that the code changes did not change the functionality?  ie. Are we confident that the same Voronoi diagram is being computed for the same input, at least for a few sets of input data?

**Comment by maxm17** [comment]
@andy-beer Hi Andy, I didn't test the output with any real data, I just used the unit tests that were already in place which had good coverage

**Comment by andy-beer** [comment]
Thanks for the clarification Max @maxm17 .  We have to be a bit careful when relying on the coverage statistics.  Those are a measure of what proportion of the lines of code have been executed by the unit tests.  The unit tests might _**or might not**_ check that the functionality is correct, ie. whether the code gets the right answer.

In the case of Delauney triangulation and the computation of Voronoi diagrams, the unit tests can only make a few very basic sanity assertions about the answers.  They do not check that the answers are correct.  This is because to do so would require the unit tests to have another implementation of the complex calculation of these functions, which is not practical.

Instead, when writing the original code, I augmented the automated testing with manual testing which involved me literally looking at the diagrams being generated. Visually a human eye can determine whether the answer is correct or not without too much trouble.

**Pull request merged**